src/coreclr/unwinder/arm64/unwinder.cpp

```diff
@@ -1689,72 +1751,35 @@ Routine Description:
         specified function. If appropriate, this should have already been
         probed.
 
-    ContextRecord - Supplies the address of a context record.
+    FunctionEntryExtended - Supplies a pointer to the variable containing
+        address of the extended function entry. It contains a non-NULL when
+        opcodes are present in .xdata.
 
-    EstablisherFrame - Supplies a pointer to a variable that receives the
-        the establisher frame pointer value.
+    ExceptionHandler - Supplies a pointer to a variable that receives
+        the exception handler routine address.
 
-    HandlerRoutine - Supplies an optional pointer to a variable that receives
-        the handler routine address.  If control did not leave the specified
-        function in either the prolog or an epilog and a handler of the
-        proper type is associated with the function, then the address of the
-        language specific exception handler is returned. Otherwise, NULL is
-        returned.
-
-    HandlerData - Supplies a pointer to a variable that receives a pointer
-        the the language handler data.
-
-    UnwindParams - Additional parameters shared with caller.
-
-    UnwindFlags - Supplies additional flags for the unwind operation.
+    ExceptionHandlerData - Supplies a pointer to a variable that receives
+        the exception handler data address.
 
 Return Value:
 
     STATUS_SUCCESS if the unwind could be completed, a failure status otherwise.
-    Unwind can only fail when validation bounds are specified.
 
 --*/
-
 {
-    ULONG AccumulatedSaveNexts;
```

Comment:
This file contains the original implementation of the unwinders for Windows unwind info. We have been trying to keep it in sync with the master copy so that it is easy to pick up bug fixes, but this refactoring seems to diverging it quite a bit. Is this refactoring necessary to achieve what you need?

cc @dotnet/dotnet-diag

Reply:
Cool, I'll give a go 👍

Reply:
Done. Thanks, Jan.